docs/main/prerequisites.md

```diff
@@ -14,7 +14,11 @@ Pi-hole is very lightweight and does not require much processing power
 Despite the name, you are not limited to running Pi-hole on a Raspberry Pi.
 Any hardware that runs one of the supported operating systems will do!
 
-### Supported Operating Systems
+### Software
+
+Pi-hole will only work on distributions utilizing [systemd](https://systemd.io/) or [sysvinit](https://www.nongnu.org/sysvinit/)! Distributions like MX Linux, Devuan, Void Linux and others will not work.
```

Comment:
The first sentence is fine. There's no need for the second sentence.

Reply:
Fixed